src/gui/guiScrollBar.cpp

```diff
@@ -364,11 +379,7 @@ s32 GUIScrollBar::getPos() const
 
 s32 GUIScrollBar::getTargetPos() const
 {
-	if (target_pos.has_value()) {
-		s32 clamped = core::s32_clamp(*target_pos, min_pos, max_pos);
```

Comment:
Are you sure this clamp can be removed?

If I disable smooth and elastic scrolling, and keep scrolling after reaching an end, then scrolling back takes time. Maybe related?

Reply:
Since this is the getTargetPos and not the true getPos code, this clamping is unneeded due to the elastic effect.